src/v2/Apps/Order/Routes/Offer/index.tsx

```diff
@@ -258,7 +258,11 @@ export class OfferRoute extends Component<OfferProps, OfferState> {
 
     const artwork = this.props.order.lineItems?.edges?.[0]?.node?.artwork
     const isInquiryCheckout = !artwork?.isPriceRange && !artwork?.price
-    const isEdition = artwork?.isEdition
+
+    // TODO: fix typescript error for displayPriceRange
+    const displayRange =
+      this.props.order.lineItems?.edges?.[0]?.node?.artworkOrEditionSet
+        ?.displayPriceRange || false
```

Comment:
Typescript error for `displayPriceRange`

Reply:
you can just `const displayRange = !!offerItem?.displayPriceRange` and then it's done

Reply:
or if you don't need this variable somewhere else I would get rid of it and use only `!!offerItem?.displayPriceRange` where you need

Reply:
Thanks! I thought there must be a shorter way to do it.